setup/lti-config.json

```diff
@@ -0,0 +1,44 @@
+{
+  "title": "Canvas LTI Redirect Tool",
+  "scopes": [],
+  "extensions": [
+      {
+          "domain": "{app-hostname}",
+          "platform": "canvas.instructure.com",
+          "settings": {
+              "platform": "canvas.instructure.com",
+              "placements": [
+                  {
+                    "default": "disabled",  
```

Comment:
Tool will disabled by default and only be available in the course navigation for users to enable in the course.